extension/packages/nextjs/app/eip-712/page.tsx

```diff
@@ -148,7 +166,7 @@ const Eip712: NextPage = () => {
           Verify (frontend)
         </button>
         <button
-          className="btn btn-primary btn-sm"
+          className={`btn btn-primary btn-sm${isLoading ? " loading" : ""}`}
```

Comment:
I think the proper way to do loading in DaisyUI v4 is using `<span className='loading loading-spinner' />` checkout this example : https://daisyui.com/components/button/#button-with-loading-spinner-and-text

I am not able to find the changelog but I remember they deprecating it suggesting the above new way to use instead while I was migrating daisUI to new version. 

Also maybe we could disbale when isLoading is true too ?

Reply:
[22a16c7](https://github.com/scaffold-eth/create-eth-extensions/pull/5/commits/22a16c764475ca4b211dd809e348880eab8f5926)